tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/tabbed_paragraph_block.html

```diff
@@ -43,8 +43,16 @@
                 <div class="tabs__panel-text">
                     {{ item.text|richtext }}
                 </div>
-                {% if item.button_text and item.button_link %}
-                    <a href="{% pageurl item.button_link %}" class="button tabs__panel-button">{{ item.button_text }}</a>
+                {% if item.button_text%}
```

Comment:
I don't think this should be wrapped with {% if item.button_text%}

Reply:
![Screenshot 2024-07-17 at 15 33 48](https://github.com/user-attachments/assets/84b4361a-05eb-4db1-8d52-5046f6122af1)

Using a button link to internal page

Reply:
![Screenshot 2024-07-17 at 15 34 45](https://github.com/user-attachments/assets/adc76b2b-9866-400a-89ee-5ef75e0324b3)

Using an external link

Reply:
Hey @nickmoreton, at mobile the block changes to be an accordion style, and you have to click on it to reveal the content (including the button) below.

I don't think you'd want a button with no text, so the check for the text seems fine to me.

Reply:
Basically, I want to check there is some text and at least one of button url or button link - django templates is a bit funny about logic and I had to nest it.

Reply:
Hi @helenb I see what you mean. 

But if it's always a requirement to enter button text if you either choose a link or enter a URL then the button text field should be a required field in the admin. Then your template logic works OK. Although, you might be able to drop the check for button text in the template as it will always be present.

Reply:
> When I tested on different screen sizes the buttons don't show on smaller screens.

This was probably harder to spot, maybe these screen shots help.

![Screenshot 2024-07-18 at 09 14 06](https://github.com/user-attachments/assets/5951fa47-137d-45db-b4db-db308f092074)

![Screenshot 2024-07-18 at 09 14 32](https://github.com/user-attachments/assets/8024aa89-9dbb-4439-a579-600f341efc70)

The button is missing when I open the First tab

Reply:
Just noting this is the only outstanding question. ^^

Reply:
The button issue at mobile should now be sorted @nickmoreton. Other issues we discussed verbally and agreed the logic is OK.

Reply:
Looks good 👍